unstructured/ingest/v2/README2.md

```diff
@@ -0,0 +1,353 @@
+# Developing V2 Connectors
+## Intro
+The Unstructured open source repo processes documents (artifacts) in a pipeline. The Source and Destination connectors sit at the front and back of the pipeline. For a visual example see the flow diagram at the bottom (link to bottom).
```

Comment:
Hmm...The "repo" doesn't process them. Perhaps "library" instead? Also, coming very soon, the open-source library won't be able to process them either, based on the move that was begun over to `unstructured-ingest`. Perhaps "Unstructured Ingest" instead?

Reply:
Suggestion: Use lowercase for names of things, here and throughout, unless it is a product name or feature name that we typically use uppercase for. For instance, "source" and "destination" instead of "Source" and "Destination."

Reply:
Don't forget the link here. Also, instead of "see the flow diagram at the bottom," provide the actual target heading name, to help readers more easily spot it at the end of this page.